snforge_std/src/cheatcodes.cairo

```diff
@@ -9,16 +9,25 @@ mod tx_info;
 mod fork;
 mod storage;
 
+/// `CheatTarget` is an enum used to designate the contracts to which a cheat should be applied.
 #[derive(Drop, Serde, PartialEq, Clone, Debug, Display)]
 enum CheatTarget {
+    /// Applies the cheatcode to all contract addresses.
     All: (),
+    /// Applies the cheatcode to the given contract address.
     One: ContractAddress,
+    /// Applies the cheatcode to each given address.
     Multiple: Array<ContractAddress>
 }
 
+/// `CheatSpan` is an enum used to specify how long the target should be cheated for.
+/// Please note that `CheatTarget::All` can only be used with `CheatSpan::Indefinite`.
```

Comment:
I don't know if this is the best place for this information 🤔

Reply:
I guess we could leave that here, since it is already present in the documentation (in the `cheat_span.md` file). Could be marked with `### Note` or `### Info`, though:
```suggestion
/// ### Note
/// `CheatTarget::All` can only be used with `CheatSpan::Indefinite`.
```

wdyt?

Reply:
Better 👍